pkg/crc/machine/machine.go

```diff
@@ -675,3 +671,33 @@ func updateSSHKeyPair(sshRunner *crcssh.SSHRunner) error {
 
 	return err
 }
+
+func configureCluster(ocConfig oc.OcConfig, sshRunner *crcssh.SSHRunner, pullSecret string) (rerr error) {
+	m := errors.MultiError{}
+	sd := systemd.NewInstanceSystemdCommander(sshRunner)
+	if _, err := sd.Start("kubelet"); err != nil {
+		return fmt.Errorf("Error starting kubelet: %s", err)
+	}
+
+	defer func() {
+		// Stop the kubelet service.
+		if _, err := sd.Stop("kubelet"); err != nil {
+			m.Collect(err)
+		}
+		if err := cluster.StopAndRemovePodsInVM(sshRunner); err != nil {
+			m.Collect(err)
+		}
+		rerr = m.ToError()
+	}()
+
+	// Update the user pull secret before kubelet start.
```

Comment:
Looks like outdated comment? The kubelet was started just before.

Reply:
@cfergeau By default the pull secret set to null string in our bundle so logically we are updating the pull secret but since on UX side if we say updating the pull secret then user might think that there is already existing pull secret so we just say `Adding user's pull secret` :)

Reply:
My problem with this comment is `before kubelet start` since the code is:
```
if _, err := sd.Start("kubelet"); err != nil {
		return fmt.Errorf("Error starting kubelet: %s", err)
}
//...
// Update the user pull secret before kubelet start.
```

Reply:
@cfergeau Ah I got it now, I will remove the `before kublet start` part from comment.